tweets/09-29-2020-triage-workflow.tweet

```diff
@@ -0,0 +1 @@
+An important announcement went out yesterday on the kuubernetes-dev mailing list about new changes to the issue triage workflow. As of Wednesday September 30th, several updates will be rolled out. Check out the email for details: https://github.com/kubernetes-sigs/contributor-tweets.git
```

Comment:
```suggestion
An important announcement went out yesterday on the kuubernetes-dev mailing list about new changes to the issue triage workflow. As of Wednesday September 30th, several updates will be rolled out. Check out the email for details: https://groups.google.com/g/kubernetes-dev/c/YcaJpsjjLKw/m/i15cLLx5CAAJ
```
Updated the link 👍

Reply:
Oh I could have just accepted your correction. Sorry about that, clipboard flub. It's corrected now!